src/games/GameHub.ts

```diff
@@ -273,14 +273,21 @@ export default class GameHub {
       throw e;
     }
 
+    const minPlayers = clazz.config;
     const cleanedPlayers = await this.cleanPlayers(players);
+    const permissions = this.getChannelPermissions(cleanedPlayers);
+    const channel = await this.createChannel(config, permissions);
+    let maybeVoiceChannel;
     const playerPing = cleanedPlayers
       .map((member) => `<@${member.user.id}>`)
       .join(" ");
 
-    const permissions = this.getChannelPermissions(cleanedPlayers);
-    const channel = await this.createChannel(config, permissions);
-    let maybeVoiceChannel;
+    if (cleanedPlayers.length < clazz.config.minPlayers) {
```

Comment:
Since you extracted minPlayers to a variable, you can use it here as well:
```suggestion
    if (cleanedPlayers.length < minPlayers) {
```